analytics/components/Jitsu.tsx

```diff
@@ -1,7 +1,31 @@
-import { context } from "deco/live.ts";
+import { Head } from "$fresh/runtime.ts";
```

Comment:
Is this a breaking change, isn't?

Reply:
Not really since this component is not used outside the apps itself.